src/geometry/manifold/cont_mdiff.lean

```diff
@@ -713,6 +713,18 @@ lemma cont_mdiff_at_iff_cont_mdiff_on_nhds {n : ℕ} :
 by simp [← cont_mdiff_within_at_univ, cont_mdiff_within_at_iff_cont_mdiff_on_nhds,
   nhds_within_univ]
 
+/-- Note: does not hold for `n = ∞`. -/
```

Comment:
Please, explain in more details: mention that we allow an infinitely smooth function at a point to be $C^n$ smooth on $U_n$, where $U_n$ becomes smaller and smaller as $n$ tends to infinity.